### DISCUSSION TITLE
Optifine compatibility?

### DISCUSSION THREAD

**Discussion by theskoopster** [Sinytra/ForgifiedFabricAPI#34] [Q&A] [answered]
I've tried using Optifabric but I can't play without disabling Optifine, do I just have to delete optifine or is there a solution?

**Comment by Su5eD** [comment, marked as answer]
OptiFine is generally incompatible with many mods, but we have much better and open source alternatives these days. Here's a list of alternatives to use on Fabric: https://lambdaurora.dev/optifine_alternatives.
